### ISSUE TITLE
C-Make ZLIB::ZLIB linking problems om Windows

### ISSUE THREAD

**Issue by Jirinrin** [Drewol/unnamed-sdvx-clone#256]
On both of my Windows machines, I've always got this error when trying to run `GenerateWin64ProjectFiles.bat`:

![image](https://user-images.githubusercontent.com/32796923/68768771-71bc2880-0623-11ea-8e18-3fc33a49c3c7.png)

I've always just fixed this by adding `find_package(ZLIB REQUIRED) on line 17 on the root `CmakeLists.txt`, but surely something must be wrong on a deeper level?
And I can't even find any problem with the CMakeLists linking...
Does anyone else recognise this problem?

**Comment by Hinara** [comment]
Did you correctly use `vcpkg install {dependencies}` before ?

**Comment by Jirinrin** [comment]
Yeah, and note how it _does_ work when I include an extra `find_package(ZLIB REQUIRED)` in the root CmakeLists.txt file

**Comment by Hinara** [comment]
So I think that FindLibraries.cmake might be not called ? as it as this line inside :/ Can you add a `message("Something")` inside to check if it is correctly called

**Comment by ASleepyCat** [comment]
I had the same problem and I fixed it by including `"-DVCPKG_TARGET_TRIPLET=x64-windows-static"` as a launch arg with CMake.

**Comment by Jirinrin** [comment]
Ah sorry Hinara, I forgot to reply ._.
But the `-DVCPKG_TARGET_TRIPLET=x64-windows-static" actually works! Thankssss now I don't have to stash this edit to my CMakeLists.txt every time I switch branches :>

**Comment by ASleepyCat** [comment]
I don't really know why specifying the triplet is only required for zlib. [The default is `x86-windows`](https://github.com/Microsoft/vcpkg/blob/master/docs/users/triplets.md#additional-remarks). You'd have to ask Drewol or someone more knowledgeable with vcpkg for the first question.

**Comment by Hinara** [comment]
Maybe zlib only accept to compile in static on your system ? 
`-DVCPKG_TARGET_TRIPLET=x64-window` might also work
It's because as said before on some computers it might try to find 32bit librairies instead of 64bit
